coralproject/talk
# [CORL-2643]: DE translation fixes

## What does this PR do?

Fixes some places where DE translations were incorrect, as pointed out in [this issue](https://github.com/coralproject/talk/issues/4035). I also found a case where `<RelativZeit></RelativZeit>` needed to be updated to `<RelativeTime></RelativeTime>` in a few translations.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

 Change your locale configuration to German. Navigate to the parts of Coral that are affected by these issues and see that all appears as it should. Specifically:
 * When replying to a comment, the `In reply to` text includes the username
 * The hidden information on a comment (check in dev tools--look for div with `Hidden-root` class for a comment in the stream) should include the relative time when the comment was created. Check a featured comment too. Also a comment in profile history.
 
## How do we deploy this PR?
